src/utils/parse-html.ts

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
Reviewers - should this be a component, rather than a util?

Reply:
I see no reason why it should be a component. It would make it more React-centric but there are no props to handle and no markup to add

Reply:
Main point of contention for me was that it returns a ReactElement, so it made me stop and ask the question. I don't think I'm particularly swung to either side, so I'm happy to keep it in the utils.